src/components/pages/TrophyRoom/RenderTrophyRoom.js

```diff
@@ -80,8 +88,9 @@ const RenderTrophyRoom = props => {
             }
           })} */}
             </div>
-          </Modal>
+          </div>
           <Divider />
+          {/* Add this to Change Your Avatar once it is implemented */}
           <h2 className="h2" onClick={showInventoryModel}>
             Inventory
```

Comment:
I think we should add a className to this inventory header with cursor: pointer to indicate on hover that it's clickable for the modal, perhaps in the future we or someone can add a hover effect as well

Reply:
I'll implement the cursor: pointer this afternoon before we merge into main. Thanks for the suggestion, totally overlooked that.

Reply:
Added the cursor and pushed to the branch.